proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
+* Tends to not commit to longer term backwards compatibility 
+
+== CN4J
+Pros
+
+* Allows for parties interested in both MPWG and CN4J/Jakarta to sign one working group participation agreement
+* Addresses IP gap in existing MP development
+* Tends to focus on stability and backwards compatibility 
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+
+Cons
+
+* Currently has a higher entry cost and overall overhead to run
+* Currently has no zero entry participation model for larger companies to sponsor specification development
+* Introduces a conflict of interest between evolution and support of enterprise customers and their previous investment in Java EE with a shared steering committee
+* Has more process around separate steering, specification and marketing committees
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
```

Comment:
Can you explain a bit more this? At the moment, in MP implementation, I feel the compatibility is not properly governed. I was looking at how to make it more formal. I think CNWG will be able to help on this.

Reply:
I agree that we probably need to tighten up our MP compatibility process a bit, but that's not the point of this bullet.  Currently, under the Jakarta EE Working Group model (CN4J), you have to be a Working Group member to get access to the Logo.  It costs some $$$ for corporations and organizations to belong to that Working Group.  The MPWG proposal is requesting a no-fee access to the Compatibility Logo.  Just prove you have passed and you can use the logo.

Reply:
that is a subtlety though that could be addresses in a single working group not a feature solved inherently by having two working groups.

Reply:
Correct, but feedback on this topic has been that such a change in position would not be acceptable to some in Jakarta.